### ISSUE TITLE
Change planning handling

### ISSUE THREAD

**Issue by KwikKill** [InsaLan/backend-insalan.fr#125]
Since #118 and #121 were merged, we need to change from using google sheet export to ics. 
This may require a new app (like cms) only to upload named file instead of only pasting the file content as a cms content.

**Comment by ShiroUsagi-san** [comment]
agree, that's in the roadmap (to allow using of images in content)
